app/repositories/application/OnlineTestRepository.scala

```diff
@@ -184,6 +186,21 @@ class OnlineTestMongoRepository(dateTime: DateTimeFactory)(implicit mongo: () =>
     collection.update(query, applicationStatusBSON, upsert = false) map ( _ => () )
   }
 
+  override def removeProgressStatuses(appId: String, progressStatuses: List[ProgressStatus]): Future[Unit] = {
```

Comment:
I thought that the whole repo is for Phase1 (bad naming). I like small repos. We have already few methods only for phase1.